server/api/users/users.controller.ts

```diff
@@ -38,3 +38,24 @@ export const getUserById: RequestHandler<{id: string}, UserProfile> = async (req
 
   res.json(user);
 };
+
+export const getUsers: RequestHandler<UserProfile[]> = async (req, res) => {
+  const users = await User.findAll({
+    attributes: [
+      'id',
+      'discord_user_id',
+      'discord_name',
+      'bio',
+      'twitter_username',
+      'linkedin_url',
+      'github_username',
+      'website',
+    ],
+  });
+
+  if (users.length === 0) {
+    throw new NotFoundError('No Users found');
+  }
```

Comment:
It's okay for there to be no users - the route can be described as "give me all the users", and "here are all the users, which is zero of them" is an okay response! I would not error here